cms/djangoapps/modulestore_migrator/tasks.py

```diff
@@ -37,8 +38,11 @@
     PublishableEntityVersion
 )
 from user_tasks.tasks import UserTask, UserTaskStatus
+from xblock.core import XBlock
+from django.utils.translation import gettext_lazy as _
 
 from common.djangoapps.split_modulestore_django.models import SplitModulestoreCourseIndex
+from common.djangoapps.util.date_utils import strftime_localized, DEFAULT_LONG_DATE_FORMAT
```

Comment:
`DEFAULT_LONG_DATE_FORMAT` doesn't include time. Should we use `DEFAULT_DATE_TIME_FORMAT` instead?
```suggestion
from common.djangoapps.util.date_utils import strftime_localized, DEFAULT_DATE_TIME_FORMAT
```